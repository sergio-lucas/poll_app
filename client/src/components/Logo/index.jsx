import React from "react";
import PropTypes from "prop-types";
import styled from "styled-components";
import logo from "../../logo.svg";

const Logo = styled.a.attrs({
  className: "navbar-brand",
})``;

function LogoView({ width }) {
  return (
    <Logo href="/">
      <img
        alt="Poll app"
        height="ds"
        src={logo}
        width={width}
      />
    </Logo>
  );
}

LogoView.propTypes = {
  width: PropTypes.number.isRequired,
};

export default LogoView;
