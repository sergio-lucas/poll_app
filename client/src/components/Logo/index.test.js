import renderer from "react-test-renderer";

import Logo from "./index";

describe("Snapshot::<Logo /> component", () => {
  it("should render the contents of the component", () => {
    const tree = renderer
      .create(<Logo />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});