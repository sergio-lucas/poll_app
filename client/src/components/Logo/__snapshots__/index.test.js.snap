// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot::<Logo /> component should render the contents of the component 1`] = `
<a
  className="sc-bdfBwQ navbar-brand"
  href="/"
>
  <img
    alt="Poll app"
    height="ds"
    src="logo.svg"
  />
</a>
`;
